backend/tests/endpoints/test_message.py

```diff
@@ -2,3 +2,45 @@
 from main import app
 
 client = TestClient(app)
+
+fake_room_id = "44523d5325dsdf"
+fake_org_id = "1223209805sdfsa902"
+fake_message_data = {
+        "sender_id": "425324132312",
+        "room_id": fake_room_id,
+        "text": "does it work well?",
+        "reactions": [],
+        "files": [],
+        "saved_by": [],
+        "is_pinned": False,
+        "is_edited": False,
+        "created_at": "2021-12-11 01:44:37.582879",
+        "threads": []
+        }
+
+def test_send_message():
+    """ test for the send message endpoint"""
```

Comment:
First of all, thanks a lot for the tests, @AI-fae. 
The external API call is not that bad. The thing is the API call can fail for many reasons and can be slow...
The purpose here is just to isolate any dependencies and focus on the main logic.
We should then mock these API calls. 
This way we will test the logic and not rely to the API behaviours (just mimicking it)

Reply:
Thank you Zxenon.
I am not familiar with mock testing. I will and implement it later in the day.

Reply:
No problem @AI-fae.
Here are some articles that might help:
- https://testdriven.io/blog/fastapi-crud/#get-routes
- https://www.patricksoftwareblog.com/monkeypatching-with-pytest/